akka-actor/src/main/scala-2.13+/akka/util/ccompat/package.scala

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2018-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.util
+
+/**
+ * INTERNAL API
+ *
+ * based on https://github.com/scala/scala-collection-compat/blob/master/compat/src/main/scala-2.13/scala/collection/compat/package.scala
+ */
+package object ccompat {
```

Comment:
The various internal api package objects can be annotated in their desugared form:

```scala
import akka.annotation.InternalApi

package foo {
  package bar {
    @InternalApi object `package`
  }
}
```

Reply:
Interesting! Not sure if it's worth it in this case, but I learned something :smile:

Reply:
Maybe not.  Maybe worth nesting this in an internal or internals package?

That's the thing about internal API we have so many tools, which ones matter and when?

* INTERNAL API comment
* `InternalApi`
* nest in an internal(s) package
* `private[akka]`